catala/cantar/cantar.md

```diff
@@ -1,3 +1,6 @@
 La pluja que cau sobre la tenda et desperta. 
-El teu company de tenda havia deixat una bossa plena de llaminadures 
-prop del teu cap, deu ser això el que t'ha fet somniar amb les parets de sucre...
\ No newline at end of file
+El teu company de tenda havia deixat una bossa plena de llaminadures prop del teu cap, deu ser això el que t'ha fet somniar amb les parets de sucre...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.